mapbox_vector_tile/polygon.py

```diff
@@ -162,38 +143,33 @@ def _polytree_node_to_shapely(node):
         children = []
 
     else:
-        # check expectations: this branch gets executed if this node is not a
-        # hole, and has no contour. in that situation we'd expect that it has
-        # no children, as it would not be possible to subtract children from
+        # Check expectations: this branch gets executed if this node is not a hole, and has no contour. In that
+        # situation we'd expect that it has no children, as it would not be possible to subtract children from
         # an empty outer contour.
         assert len(children) == 0
 
-    return (polygons, children)
+    return polygons, children
 
 
 def _polytree_to_shapely(tree):
     polygons, children = _polytree_node_to_shapely(tree)
 
-    # expect no left over children - should all be incorporated into polygons
-    # by the time recursion returns to the root.
+    # expect no left over children - should all be incorporated into polygons by the time recursion returns to the root.
     assert len(children) == 0
 
-    union = cascaded_union(polygons)
+    union = unary_union(polygons)
```

Comment:
:bow:

Reply:
The `cascaded_union` function is deprecated: https://shapely.readthedocs.io/en/stable/manual.html#shapely.ops.cascaded_union